### PR TITLE
Swap the position of opening customer support chat and making phone calls to customer support

### LINKED ISSUE #42: 联系电话改为微信客服
1.  首页联系电话改为微信客服，用户点击后，跳转到微信客服。 首页跳转 https://work.weixin.qq.com/kfid/kfc5fdb2e0a1f297753?sence=mp-bbhj

2. "我的" ->   "联系我们" , 点击后触达 客服电话

### PR DESCRIPTION
**Pull request by jerry-shao**
Swap the position of opening customer support chat and making phone calls to customer support

### CODE REVIEW

File: src/components/Contact.vue [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-    handleClick() {
-      console.log("handleClick");
-    },
-    getPhoneNum() {
-      console.log("getPhoneNum");
-      wx.makePhoneCall({
-        phoneNumber: "0435-3338090",
+    handleOpenCustomerServiceChat() {
+      Taro.openCustomerServiceChat({
+        extInfo: { url: "https://work.weixin.qq.com/kfid/kfc02343d9ba414880a" },
```
Comment (deancn) [inline src/components/Contact.vue]: 正式的地址跳转地址是： https://work.weixin.qq.com/kfid/kfc5fdb2e0a1f297753?sence=mp-bbhj

File: src/components/Contact.vue [hunk 2]
```diff
@@ -1,31 +1,37 @@
 <template>
-  <AtButton class="connect" @click="getPhoneNum">
-    <image
-      class="img"
-      src="https://wechat-1251018873.file.myqcloud.com/images/phone.png"
-    />
-    <view class="text">0435-3338090</view>
+  <AtButton class="connect" @click="handleOpenCustomerServiceChat">
+    <view class="text">微信客服</view>
```
Comment (deancn) [inline src/components/Contact.vue:3]: 字号调整和title 一样大吧

Comment (jerry-shao) [reply src/components/Contact.vue:3]: Done

**Comment by deancn** [comment]
lgtm

**Pull request merged**